Model/updateComment.php

```diff
@@ -0,0 +1,53 @@
+<script src="../class/updateComment.js"></script>
+
+<?php
+  session_start();
+  require('DbConnection.php');
+  $db = new DbConnection();
+  // If requested for update and postId and uuserId is set and comment is not empty then add the comment to database.
+  if (isset($_REQUEST['update']) && isset($_REQUEST['pi']) && isset($_REQUEST['ui'])) {
+    if ($_REQUEST['content'] != "")
```

Comment:
is nested if required here?

Reply:
No, removed and added in one line using &&